server/src/graphql-types/EventUser.ts

```diff
@@ -0,0 +1,51 @@
+import { Field, ObjectType } from 'type-graphql';
+import { BaseObject } from './BaseObject';
+import { User } from '.';
+
+@ObjectType()
+export class Rsvp extends BaseObject {
+  @Field(() => Date)
+  updated_at: Date;
+
+  @Field(() => String)
+  name: string;
+}
```

Comment:
Nitpick: since this corresponds to a db table, I think this belongs in its own file.